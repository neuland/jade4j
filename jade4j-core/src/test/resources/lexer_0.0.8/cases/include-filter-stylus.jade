style(type="text/css")
  include:styl some.styl
