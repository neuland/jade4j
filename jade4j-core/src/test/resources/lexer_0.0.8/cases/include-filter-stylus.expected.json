{"type":"tag","line":1,"val":"style","selfClosing":false}
{"type":"attrs","line":1,"attrs":[{"name":"type","val":"\"text/css\"","escaped":true}]}
{"type":"indent","line":2,"val":2}
{"type":"include","line":2,"val":"some.styl","filter":"styl","attrs":null}
{"type":"outdent","line":3}
{"type":"eos","line":3}